gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/
+.php_cs.cache
```

Comment:
For [PHP-CS-Fixer v3](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#rename-of-files)
```suggestion
.php_cs.cache
.php-cs-fixer.cache
```